test/parallel/test-timers-now.js

```diff
@@ -1,11 +1,31 @@
+// Flags: --expose-internals --no-warnings --allow-natives-syntax
 'use strict';
-// Flags: --expose-internals
 
-require('../common');
-const assert = require('assert');
+const common = require('../common');
+const assert = require('node:assert');
 const { internalBinding } = require('internal/test/binding');
 const binding = internalBinding('timers');
 
 // Return value of getLibuvNow() should easily fit in a SMI after start-up.
 // We need to use the binding as the receiver for fast API calls.
 assert(binding.getLibuvNow() < 0x3ffffff);
+
+{
+  // V8 Fast API
+  function optimized() {
+    return binding.getLibuvNow();
+  }
+  function testFastPaths() {
+    assert(binding.getLibuvNow() < 0x3ffffff);
```

Comment:
This should not call `optmized` function?

Reply:
not necessarily. on line 30 we already check if optimized function is called or not.

Reply:
Maybe it works, but it's weird to opmitize a function but not call it in the test.